test/imageext_test.py

```diff
@@ -37,7 +36,6 @@ def test_save_bad_filename(self):
         u = u"a\x00b\x00c.png"
         self.assertRaises(pygame.error, save_extended, im, u)
 
-    @expected_error(TypeError)
     def test_load_bad_filename(self):
         u = u"a\x00b\x00c.png"
         self.assertRaises(pygame.error, load_extended, u)
```

Comment:
They do raise an error. They now raise a error SDL_Error from the load code, instead of raising a TypeError earlier, which matches pygame's behaviour.